src/js/base/js-numbers.js

```diff
@@ -1201,16 +1201,44 @@ define("pyret-base/js/js-numbers", function() {
       return bnRemainder.call(m, n);
     });
 
+  var splitIntintoMantissaExpt = function(s) {
+    var str = s.toString();
+    var c0 = str[0];
+    var sign = '';
+
+
+    if (c0 === '-' || c0 === '+') {
+      str = str.substring(1);
+      if (c0 === '-') {
+        sign = '-';
+      }
+    }
+
+    var d0 = str[0];
+    str = str.substring(1);
+
+    var mantissa = Number(sign + d0 + '.' + str);
```

Comment:
Oh, it is an int, based on the name (which I misread as "split into mantissa expt") -- need another capital letter in there...